app/models/reviewable_flagged_post.rb

```diff
@@ -72,7 +72,7 @@ def build_actions(actions, guardian, args)
 
     build_action(actions, :ignore, icon: 'external-link-alt')
 
-    if potential_spam? && guardian.can_delete_all_posts?(target_created_by)
+    if potential_spam? && guardian.can_delete_user?(target_created_by)
```

Comment:
Could you explain this change please? It doesn’t have any impact on any spec?

Reply:
Thought there was a test but there wasn't. I added some now!

`can_delete_all_posts?` could return true in scenarios where the user cannot be deleted. Switched to `can_delete_user?` which is what the `UserDestroyer` does.